lib/diagnostic.rb

```diff
@@ -38,31 +38,31 @@
 
 # define your method here
 
-Diagnostic.predicate = # call your method here
+Diagnostic.predicate = # &:odd?
 
```

Comment:
This works, but we were looking for a new method defined conventionally for a predicate, such as:

```
def num_odd? (n)
  # something with n
end
```